app/code/community/Sendit/Bliskapaczka/Block/Adminhtml/Order/Grid.php

```diff
@@ -199,4 +208,24 @@ public function getGridUrl()
     {
         return $this->getUrl('*/*/grid', array('_current'=>true));
     }
+
+    /**
+     * @throws Exception
+     */
+    protected function prepareFirstAndLastNameColumns()
```

Comment:
W Magento metody prywatne mają nazwy zaczynające się od _, w tym przypadku powinno to być _prepareFirstAndLastNameColumns

Reply:
poprawione. Dzięki za info